tox.ini

```diff
@@ -0,0 +1,6 @@
+[tox]
+envlist = py27,py34,py35,py36
```

Comment:
@jdufresne About `py37`, I'm not sure because it's not present in the [setup.py](https://github.com/paramiko/paramiko/blob/master/setup.py#L72) of the project.